CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.5.0 (unreleased)
```

Comment:
Please review the changelog. `singletons` does the automatic detection now. Also, add items for `expectEvent` and `configure` (`provider` vs `web3`). Am I missing something else?